src/components/Detail.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';

```

Comment:
### Kindly stick with the design guidelines(There are two colors for design one red and one blue) for the `details` page: 
- [ ] Please change `colors`, `background-colors`, and `alignment` to match the design guidelines.
- [ ] The header of the details page should contain the image on the left and the info on the right.
- [ ] Below the header there is a list of items.

| Your design | Design guidelines |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/170548923-69e17fe6-7bc8-4e52-82a4-3b1436af4e50.png) | ![image](https://user-images.githubusercontent.com/24703171/170548658-6f76d159-db6c-4be9-85ec-c626f937dd0d.png) |